roles/test_operator/README.md

```diff
@@ -21,8 +20,20 @@ Execute tests via the [test-operator](https://openstack-k8s-operators.github.io/
 * `cifmw_test_operator_node_selector`: (Dict) `nodeSelector` value that is applied to all pods spawned by the test-operator and to the test-operator-controller-manager and test-operator-logs pods. Default value: `{}`
 * `cifmw_test_operator_storage_class_prefix`: (String) Prefix for `storageClass` in generated Tempest CRD file. Defaults to `"lvms-"` only if `cifmw_use_lvms` is True, otherwise it defaults to `""`. The prefix is prepended to the `cifmw_test_operator_storage_class`. It is not recommended to override this value, instead set `cifmw_use_lvms` True or False.
 * `cifmw_test_operator_storage_class`: (String) Value for `storageClass` in generated Tempest or Tobiko CRD file. Defaults to `"lvms-local-storage"` only if `cifmw_use_lvms` is True, otherwise it defaults to `"local-storage"`.
-* `cifmw_test_operator_delete_logs_pod`: (Boolean) Delete tempest log pod created by the role at the end of the testing. Default value: `false`.
+* `cifmw_test_operator_delete_logs_pod`: (Boolean) Delete tempest log pod created by the role at the end of the testing. Default value: `true`.
 * `cifmw_test_operator_privileged`: (Boolean) Spawn the test pods with `allowPrivilegedEscalation: true` and default linux capabilities. This is required for certain test-operator functionalities to work properly (e.g.: `extraRPMs`, certain set of tobiko tests). Default value: `true`
+* `cifmw_test_operator_stages`: (List) List of dictionaries defining the stages that should be used in the test operator role. List items options are:
+  * `name`: (String) The name of the stage. The name must be unique.
+  * `type`: (String) The framework name you would like to call, currently the   options are: tempest, ansibletest, horizontest, tobiko.
+  * `test_file`: (String) Path to the file used for testing, this file should contain the testing params for this stage. Default value: `{{ ansible_user_dir }}/ci-framework-data/artifacts/ansible-vars.yml`
```

Comment:
@lpiwowar the reason I wanted it to be a file was to be able to avoid having to specify all the parameters inside the stages structure which can make it very large and hard to manage. I can add an additional interface if you think appropriate but I would prefer to keep the test var file. Please refer to this example https://gitlab.cee.redhat.com/ci-framework/ci-framework-jobs/-/merge_requests/1112/diffs#6a0b57992826d55c067fdc35db4ceb137f0180a2_23_20 .

Reply:
In the provided link most of the space of the stage definition is taken by the hooks. Looking at the stage definition I want to see what tests are being executed there and with what configuration. To me this starts to feel too nested. Job definition -> vars file -> tests file. This would mean that for each stage there will be with high probability a new test file. I'm not sure whether we want to go this way.

I would prefer to go with `test_vars`. I guess we need somebody to decide what is best. 

But one way or the other I would rename the var to `test_vars` or `test_vars_file`.

Reply:
@lpiwowar please note that assuming you want to run several tempest stages you would have to specify ALL (different) variables in the stages definition. maybe it wasn't clear in the reference I shared but it runs completely different tempest stages 3 times having to specify all the vars in the stage definition would make it hard to manage. I believe having test variables in different files should simplify it and make it easier to manage and review. I don't mint having both but not having the file one would complicate things for some jobs IMHO.

Reply:
@eshulman2, ok, it make sense. In that case if everyone agrees, can we do these two things?

- [ ] Add a parameter `test_vars` which can be used in cases when the test definition is not lengthy.
- [ ] Rename the `test_file` parameter to `test_vars_file`.